### ISSUE TITLE
Validation Results Docs references file function - should be open function

### ISSUE THREAD

**Issue by daccorti** [fivetran/great_expectations#3630]
**Describe the bug**
In the [validation results section](https://github.com/great-expectations/great_expectations/blob/develop/docs/reference/validation.md#validation-results) the example provides code that uses the file function to open your expectations json file. Super small bug, but should probably switch to the `open ` function as I believe that is what replaced `file` in Python 3.

**To Reproduce**
```
import json 
import great_expectations as ge

my_expectation_suite = json.load(file("my_titanic_expectations.json"))
my_df = ge.read_csv(
  "./tests/examples/titanic.csv", 
  expectation_suite=my_expectation_suite
)
my_df.validate()
```

**Expected behavior**

**Environment (please complete the following information):**
 - Operating System: Windows 10
 - Great Expectations Version:  version 0.13.40

**Additional context**
N/A

**Comment by talagluck** [comment]
Thanks for raising this, @daccorti! This does appear to  be bug - would you be interested in making a contribution here? We're happy to offer any guidance/support you may need.

**Comment by austiezr** [comment]
Closed by #3632
